src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/check_health.py

```diff
@@ -153,7 +157,17 @@ def _get_helm_version(ignore_errors):
 
     if warning:
         print_warning(warning)
-    print("Helm version:\n{}".format(output))
+
+    # Retrieve the helm version if regex pattern is found
+    match_obj = HELM_VERSION_REGEX.search(output)
+    if match_obj:
+        output = match_obj.group(1)
+
+    print("Helm version: {}".format(output))
+
+    # Display an error message if the current helm version < min required version
+    if match_obj and StrictVersion(output) < StrictVersion(MIN_HELM_VERSION):
+        print_error("Current Helm version is obsolete. Please upgrade to at least {}".format(MIN_HELM_VERSION))
```

Comment:
Don't use `print_error` here, use `_handle_error` function. Try adapting the error at `_errors.py` or, if needed, creating a new error over there and import it over here.
The reason for that is because we want to direct customers to a library where they can search the errors by their codes.

Reply:
I think that updating the error message for HELM_VERSION_ERROR is the ideal, considering that the error is pretty close already XD https://docs.microsoft.com/en-us/azure/container-registry/container-registry-faq#helm_version_error

Reply:
@ni-reis 

Should we use the error message from the website then?

Reply:
No, just use the error library in the module. Something like the other checks that the command does.